pytest_splunk_addon/standard_lib/utilities/create_new_eventgen.py

```diff
@@ -0,0 +1,245 @@
+import argparse
+import re
+import os
+
+from splunk_appinspect import App
+from .mapping import FIELD_MAPPING, FILE_MAPPING
+
+
+class UpdateEventgen():
+    """ Update eventgen file """
+
+    def __init__(self, addon_path):
+        self._app = App(addon_path, python_analyzer_enable=False)
+        self._eventgen = None
+        self.path_to_samples = os.path.join(addon_path, "samples")
+
+    @property
+    def eventgen(self):
+        try:
+            if not self._eventgen:
+                self._eventgen = self._app.get_config("eventgen.conf")
+            return self._eventgen
+        except OSError:
+            raise Exception("Eventgen.conf not found")
+            return None
+
+    def get_eventgen_stanzas(self):
```

Comment:
Are we using the same `get_eventgen_stanzas` method from EventgenParser or different?